### ISSUE TITLE
Rate Limiting

### ISSUE THREAD

**Issue by codezninja** [jamietsao/slack-multi-channel-invite#11]
So I'm getting a rate limiting error when trying to add one user to one channel on my workspace. It could be that's its a large workspace. Here's the error message

```
{"ok":false,"error":"ratelimited"}
panic: Non-200 status code (429)

goroutine 1 [running]:
main.main()
        /Users/cdt/go/pkg/mod/github.com/jamietsao/slack-multi-channel-invite@v1.2.0/main.go:124 +0xa7f
```

**Comment by jamietsao** [comment]
Yeah based on the line of code where the error is occurring, rate limit is being hit while retrieving all the channels.  Is this happening every time you run the command or only when you run it many times in a short period?
